README.md

```diff
@@ -2,208 +2,394 @@
 
 ## Learning Goals
 
-- Explain in general terms what the execution context is.
-- Describe the difference between global- and function-scoped code.
-- Understand how block scoping affects variables declared with `let` and `const`.
+- Define scope as a programming concept.
+- Describe the three levels of scope in JavaScript.
+- Examine the exceptions to the scope rules.
 
 ## Introduction
 
-_Scope_ is, in short, the concept of **where something is available**. In the
-case of JavaScript, it has to do with where declared variables and methods are
-available within our code.
+The more complex your code becomes, the more important it is to pay attention to
+scope. So, what exactly is it? _Scope_ is the concept of **where something is
+available**. In the case of JavaScript, it has to do with where variables and
+functions are accessible within our code.
 
-Scope is a ubiquitous concept in programming and one of the most misunderstood
-principles in JavaScript, frustrating even seasoned engineers. Not understanding
-how scope works will lead to pain.
+Scope is not something fully tangible, there is no specific syntax to it. As a
+result, it can be a difficult concept to grasp. However, scope affects
+everything you write. All code you have written thus far has used scope in some
+way, even without you knowing. Consequently, it is important that you understand
```

Comment:
```suggestion
way, even without you knowing. And, since all the code you write in the _future_ will also use scope, it is important that you understand
```